src/components/SmallButton.js

```diff
@@ -20,12 +21,12 @@ const StyledButton = styled.button`
   line-height: 0;
   width: 20px;
   height: 20px;
-  border: 2px red solid;
+  border: 2px ${prop => prop.color} solid;
   border-radius: 50%;
-  color: red;
+  color: ${prop => prop.color};
```

Comment:
You could think about making the color optional and providing a fallback value so that you only have to style the color of single buttons that are supposed to be different. E.g.: `color: ${props => props.inputColor || "palevioletred"};`

Reply:
good point!